gensim/models/rpmodel.py

```diff
@@ -5,41 +5,54 @@
 # Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
 
 
-import logging
+"""
+Objects of this class allow building and maintaining a model for Random Projections
+(also known as Random Indexing). 
 
-import numpy as np
+For theoretical background on RP, see:   Kanerva et al.: "Random indexing of text samples for Latent Semantic Analysis."
```

Comment:
Need to add concrete link to paper, something like this - https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/parsing/porter.py#L6